### ISSUE TITLE
"insert called without data to insert"

### ISSUE THREAD

**Issue by rjhilgefort** [miguelbermudez/eslint-docset#2]
Attempting to generate via instructions outlined in readme- from a release download. I'm getting the following error:

![screenshot 2016-11-23 17 18 45](https://cloud.githubusercontent.com/assets/2034823/20580581/214b8bd6-b1a1-11e6-86ba-87808070c91a.png)

**Comment by JasonSooter** [comment]
@rjhilgefort I received the same error. However, all Eslint Rules docs imported into Dash with no errors. Just and FYI

**Comment by rjhilgefort** [comment]
> @rjhilgefort I received the same error. However, all Eslint Rules docs imported into Dash with no errors. Just and FYI

@JasonSooter Hmmm, I didn't have the same luck. I'll try again as it's been a while. Thanks for the heads up.

**Comment by JasonSooter** [comment]
@rjhilgefort Here's the working docset file it created for me ;-)

[eslint-docs.docset.zip](https://github.com/miguelbermudez/eslint-docset/files/755507/eslint-docs.docset.zip)

**Comment by rjhilgefort** [comment]
@JasonSooter Thanks!

**Comment by rjhilgefort** [comment]
@JasonSooter Oh, it's empty?

**Comment by JasonSooter** [comment]
![eslint-docs_docset_and__insert_called_without_data_to_insert__ _issue__2_ _miguelbermudez_eslint-docset_and__mysql_5_5_54-0_deb7u1__magento_database_uwqfmkhxex_aw_blog_and_search_results_for___lookbook_](https://cloud.githubusercontent.com/assets/7215306/22661935/d438dc7a-ec6c-11e6-94a9-141dfaf9fc80.png)

I show it to have contents. Opening it up in Dash should look like this:
![dash](https://cloud.githubusercontent.com/assets/7215306/22661985/02bd5c60-ec6d-11e6-9914-24a56e755455.png)

**Comment by rjhilgefort** [comment]
@JasonSooter Ah, yep. I had to remove the one I generated to see the "home" html when selecting the docset. I also was expecting to be able to search for the rules- that's the primary advantage of a docset (more so than the offline access). As well as have the rules listen in the side bar. It looks like all the rules have their own HTML entry so it may just be that the docset needs to be organized differently. Anyways, thanks! Hopefully this gets some love at some point.

![image](https://cloud.githubusercontent.com/assets/2034823/22662218/3bb8ce1a-ec76-11e6-92f9-1717dec2e19b.png)

**Comment by JasonSooter** [comment]
Yep. Not perfect. I'll see if I can figure it out. I built one for RethinkDB so maybe I can use the same pattern
